ParisBNF/abb/BNFabb150.xml

```diff
@@ -28,6 +28,140 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        </layout>
+                      </layoutDesc>
+                    </objectDesc>
+                    <handDesc>
+                      <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">18th-century script</seg> 
+                      </handNote>
+                    </handDesc>
+                    <additions>
+                      <list>
+                        <item xml:id="a1">
+                          <locus target="#1"/>
+                          <desc type="GuestText">An incomplete prayer against eye and head pain. <bibl>
+                            <ptr target="bm:Chaine1912Catalogue"/>
+                            <citedRange unit="page">91</citedRange>
+                            <citedRange unit="number">150</citedRange>
+                          </bibl> defined it as 'two magic formulas'.</desc>
+                          <q xml:lang="gez">ይትባረክ፡ እግዚአብሔር፡ ጸሎተ፡ በእንተ፡ ሕምም፡ አይን፡ ወርዕስ፡ መዕመን፡ ኦሪት፡ ወመድኃኒት፡ ዘበምራቅከ፡ ትከሥት፡ አይንተ፡ ዕዉራን፡</q>
+                          <note>According to <bibl>
+                            <ptr target="bm:Abbadie1859Cat"/>
+                            <citedRange unit="page">164</citedRange>
+                          </bibl>, there were two detached folia containing another Christian prayer, which were already missing already by the time of <bibl>
+                            <ptr target="bm:ContiRossini1914Notice"/>
+                            <citedRange unit="page">216</citedRange>
+                          </bibl>.</note>
+                        </item>
+                        <item xml:id="e1">
+                          <desc type="StampExlibris">
+                            Shelfmark of the <placeName ref="INS0303BNF"/> glued onto the front board.
+                          </desc>
+                        </item>
+                        <item xml:id="e3">
+                          <locus target="#2r #48r"/>
+                          <desc type="StampExlibris">A circle stamp, with the name
+                            <persName ref="PRS1071dAbbadi" role="owner"/> in it.</desc>
+                        </item>
+                     </list>
+                    </additions>
+                    <bindingDesc>
+                      <binding xml:id="true">
+                        <decoNote xml:id="b1">The front is a wooden board consisting of two pieces. The back board is either invisible in the images or is missing.</decoNote>
+                      </binding>
+                    </bindingDesc>
+                  </physDesc>
+                  <history>
+                    <origin>
+                      <origDate notBefore="1700" notAfter="1800" source="lettering">18th century</origDate>.  This manuscript was collected by <persName
+                        ref="PRS1071dAbbadi" role="owner"/> in Ethiopia.
+                    </origin>
+                    <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
```

Comment:
the BnF

Reply:
It is encoded, why should I repeat it? 'In ' refers to the year

Reply:
I think the article is needed linguistically, but who knows